.changeset/sour-pianos-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pantheon-systems/drupal-kit": minor
+---
+
+[drupal-kit, next-drupal] Upgrade DrupalState dependency to 3.1.0 0 db-3479
```

Comment:
```suggestion
[drupal-kit] Upgrade DrupalState dependency to 3.1.0
```
Since there is no actual change to the `next-drupal-starter` for this change, we can leave it out of this changeset.
If there was a change to the starter, I would prefer it in its own changeset so the automated CHANGELOGs will reflect each project accurately.